gax/test/test-application/src/index.ts

```diff
@@ -469,6 +470,50 @@ async function testEchoErrorWithTimeout(client: SequenceServiceClient) {
   }
 }
 
+async function testErrorDetailsWithTimeout(client: SequenceServiceClient) {
+  try {
+    const backoffSettings = createBackoffSettings(
+        100,
+        1.2,
+        1000,
+        null,
+        1.5,
+        3000,
+        1,
+    );
+    const retryOptions = new RetryOptions([14, 4], backoffSettings);
+
+    const settings = {
+      retry: retryOptions,
+    };
+
+    client.initialize();
+
+    const request = createSequenceRequestFactory(
+        [
+          Status.UNAVAILABLE, // Error code 14
+          Status.UNAVAILABLE,
+          Status.UNAVAILABLE,
+          Status.UNAVAILABLE,
+        ],
+        [0.1, 0.1, 0.1, 0.1],
+    );
+
+    const response = await client.createSequence(request);
+    const sequence = response[0];
+
+    const attemptRequest =
+        new protos.google.showcase.v1beta1.AttemptSequenceRequest();
+    attemptRequest.name = sequence.name!;
+
+    await client.attemptSequence(attemptRequest, settings);
+    assert.fail('The operation should have failed')
+  } catch (e) {
+    const expectedMessage = 'Total timeout of API google.showcase.v1beta1.SequenceService exceeded 1 milliseconds retrying error Error: 14 UNAVAILABLE: 14  before any response was received.';
```

Comment:
Maybe you should try the test with different errors so that you can test your new functionality? Assuming you want something more than just the current error message.

Reply:
In this PR it is the unit test checking for the `Previous errors` string that is more critical. My suspicion is that this test doesn't even test the code path where the source code changes are. I think it targets streaming calls, but it's nice to have anyway.

Reply:
I ran an experiment in the other PR and have verified in the test run at https://github.com/googleapis/gax-nodejs/actions/runs/14716048863/job/41299549173?pr=1743 that the previous errors are getting included in the error message.